migrate-from-tidb-to-mysql.md

```diff
@@ -156,12 +162,15 @@ After setting up the environment, you can use [Dumpling](/dumpling-overview.md)
 
 3. Enable GC.
 
-    In incremental migration using TiCDC, GC only removes history data that is replicated. Therefore, after creating a changefeed, you need to run the following command to enable GC. For details, see [What is the complete behavior of TiCDC garbage collection (GC) safepoint](/ticdc/ticdc-faq.md#what-is-the-complete-behavior-of-ticdc-garbage-collection-gc-safepoint). 
+    In incremental migration using TiCDC, GC only removes history data that is replicated. Therefore, after creating a changefeed, you need to run the following command to enable GC. For details, see [What is the complete behavior of TiCDC garbage collection (GC) safepoint](/ticdc/ticdc-faq.md#what-is-the-complete-behavior-of-ticdc-garbage-collection-gc-safepoint).
 
     ```sql
     MySQL [test]> SET GLOBAL tidb_gc_enable=TRUE;
     Query OK, 0 rows affected (0.01 sec)
     MySQL [test]> SELECT @@global.tidb_gc_enable;
```

Comment:
````suggestion
    To enable GC, run the following command:
    
    ```sql
    SET GLOBAL tidb_gc_enable=TRUE;
    ```

    ```
    Query OK, 0 rows affected (0.01 sec)
    ```

    To verify that the change takes effect, query the value of `tidb_gc_enable`:

    ```sql
    SELECT @@global.tidb_gc_enable;
````

Reply:
fixed in 5c8c22c